cmd/meroxa/global/client.go

```diff
@@ -17,116 +17,51 @@ limitations under the License.
 package global
 
 import (
-	"bytes"
-	"encoding/json"
-	"errors"
 	"fmt"
-	"io/ioutil"
-	"net/http"
-	"time"
+	"os"
 
-	"github.com/dgrijalva/jwt-go"
 	"github.com/meroxa/meroxa-go"
-	"github.com/spf13/viper"
+	"github.com/meroxa/meroxa-go/auth"
+	"golang.org/x/oauth2"
 )
 
 const (
 	clientID = "2VC9z0ZxtzTcQLDNygeEELV3lYFRZwpb"
-	domain   = "auth.meroxa.io"
 )
 
 func NewClient() (*meroxa.Client, error) {
-	accessToken, err := getAccessToken(Config)
-	if err != nil {
-		return nil, err
+	options := []meroxa.Option{
+		meroxa.WithUserAgent(fmt.Sprintf("Meroxa CLI %s", Version)),
```

Comment:
Where is `Version` being set?

Reply:
It's the very first thing that happens in main :) https://github.com/meroxa/cli/blob/bb6889750b2c060d7956c0f8aff08407f5d63305/cmd/meroxa/main.go#L26

We still rely on go-releaser to inject the version for us. We still need to figure out how to get the version when generating documentation. I've got some ideas for that but I didn't act on them yet, I'm not convinced by any particular approach, I'm curious what you think though 🙂

Reply:
@lovromazgon what I had in mind is to update the version as part of our tagging process. Leverage GitHub actions if possible. I haven't written down the steps anywhere yet, but that was roughly my idea.